### PR TITLE
SNHUT-4306 : Add Experiences from Search Results to Experience Drawer

### PR DESCRIPTION
**Pull request by mk-sada**
### Description

- Continuation of https://github.com/snhuproduct/turbine/pull/293

SNHUT-4306 : Add Experiences from Search Results to Experience Drawer

Take a pull from SNHUT-3104-PLA-search-experience-V3 this branch and continue worked.
also take pull from SNHUT-2778-drawer-task to finish this task.

#### Screenshots

Screenshot 2023-03-22 at 6 17 14 AM
Screenshot 2023-03-22 at 6 17 26 AM
Screenshot 2023-03-22 at 6 17 32 AM
Screenshot 2023-03-22 at 6 17 37 AM
Screenshot 2023-03-22 at 6 23 00 AM

| Original | Updated |
| -------- | ------- |
|          |         |

### Other Details

- [JIRA Ticket](https://collectiveshift.atlassian.net/browse/SNHUT-4306)

#### Checklist

- [x] **Assumptions of User Story met (including acceptance criteria)**
- [x] **All new necessary Unit tests were CREATED** to promote code coverage
- [x] **All required unit tests are PASSING**
- [x] The proposed functionality was tested manually locally, and it's working fine.
- [x] PR was properly reviewed by a team member (at least 1 approval required to merge!)
- [x] Unnecessary console.logs AND/OR comments were removed
- [x] Feature ok-ed by UX designer (If needed - not for back end components) and Product Owner
- [x] Ensure configuration and/or build changes documented

### CODE REVIEW

File: microservices/turbine-admin/yarn.lock [hunk 1]
```diff
@@ -6073,7 +6073,7 @@
     "color-name" "1.1.3"
 
 "color-convert@^2.0.1":
```
Comment (shafi-sadaindia) [inline microservices/turbine-admin/yarn.lock:6075]: Don't push this file

File: microservices/turbine-admin/apps/toboggan-app/src/app/prior-experiences/pages/prior-experiences-main-page/prior-experiences-main-page.component.ts [hunk 2]
```diff
@@ -1,9 +1,101 @@
...
+  clearExperience(){
+   // alert("cleared");
+   this.priorExperiencesService.clearAddedExperience();
+  }
+
+  handleDropDownAction(action: CardInfo){
+    if(action.action === "edit"){
+      //edit function
+    }
+    if(action.action === "remove"){
```
[the remark this answers is not in the document]
Comment (hilal-sadaindia) [reply microservices/turbine-admin/apps/toboggan-app/src/app/prior-experiences/pages/prior-experiences-main-page/prior-experiences-main-page.component.ts]: updated

File: microservices/turbine-admin/libs/toboggan-common/src/lib/toboggan-common.ts [hunk 3]
```diff
@@ -334,6 +334,69 @@ export interface IStaffProfile {
   lastModifiedTime?: string;
 }
 
+export declare enum tagStateEnum {
+  Recommended = 'is-recommended',
```
Comment (shafi-sadaindia) [inline microservices/turbine-admin/libs/toboggan-common/src/lib/toboggan-common.ts:338]: Change format to camelCase

Comment (hilal-sadaindia) [reply microservices/turbine-admin/libs/toboggan-common/src/lib/toboggan-common.ts]: updated

**Review by shafi-sadaindia — APPROVED** [review]
LGTM

**Pull request merged**